tikv/cop_handler.go

```diff
@@ -67,8 +69,8 @@ func (svr *Server) handleCopDAGRequest(reqCtx *requestCtx, req *coprocessor.Requ
 		chunks = appendRow(chunks, data, rowCnt)
 		rowCnt++
 	}
-	warnings := dagCtx.evalCtx.sc.GetWarnings()
-	return buildResp(chunks, e.Counts(), err, warnings, time.Since(startTime))
+	//warnings := dagCtx.evalCtx.sc.GetWarnings()
+	return buildResp(chunks, e.Counts(), err, nil, time.Since(startTime))
```

Comment:
We can change the `buildResp` `warnings` arg type as
```
func buildResp(chunks []tipb.Chunk, counts []int64, err error, warnings []stmtctx.SQLWarn, dur time.Duration) *coprocessor.Response {
```

Reply:
and change append warnings as
```
selResp.Warnings = append(selResp.Warnings, toPBError(warnings[i].Err))
```

Reply:
Thank you for your comments. Addressed.